pkg/api/apis/operators/v1alpha1/subscription_types.go

```diff
@@ -76,6 +76,14 @@ type SubscriptionConfig struct {
 	// +patchStrategy=merge
 	// +optional
 	Env []corev1.EnvVar `json:"env,omitempty" patchMergeKey:"name" patchStrategy:"merge"`
+	
+	// List of Volumes to set in the podSpec.
+	// +optional
+	Volumes []corev1.Volume `json:"volumes,omitempty"`
```

Comment:
Has the CRD schema been updated to match?

Reply:
The CRD will be updated in an upcoming change to this PR.

Reply:
The CRD will be updated in the next version of the PR.